scripts/upgrade.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/bash
+
+## Declare variables
+# VARS:
+DEFAULT_PATH="/mnt/siriuschain/"
+SIRIUS_CHAIN_VERSION="release-v0.9.0"
+P2P_PACKAGE="public-mainnet-peer-package-$SIRIUS_CHAIN_VERSION.tar.gz"
+DOWNLOAD_URL="https://github.com/proximax-storage/xpx-mainnet-chain-onboarding/releases/download/$SIRIUS_CHAIN_VERSION/$P2P_PACKAGE"
+
+## Prompt
+# Base installation directory
+printf "Enter base installation directory [default: $DEFAULT_PATH]:" 
+read base_dir
+
+if [ -z "$base_dir" ]; then
+  echo "empty.  setting base directory to default $DEFAULT_PATH"
+  base_dir=$DEFAULT_PATH
+fi
+
+
+cd $base_dir/public-mainnet-peer-package
+
+# verify whether there is an existing installation
+# TODO
+
+# retrieve existing data
+
+cd resources
+node_host=$(sed -n '/^host\s*=\s*/{s/^host\s*=\s*//;p}' config-node.properties)
+friendly_name=$(sed -n '/^friendlyName\s*=\s*/{s/^friendlyName\s*=\s*//;p}' config-node.properties)
+boot_key=$(sed -n '/^bootKey\s*=\s*/{s/^bootKey\s*=\s*//;p}' config-user.properties)
+harvest_key=$(sed -n '/^harvestKey\s*=\s*/{s/^harvestKey\s*=\s*//;p}' config-harvesting.properties)
+
+echo Existing Configuration:
+echo "Friendly Name is $friendly_name"
+echo "Host is $node_host"
+echo "BootKey is $boot_key"
+echo "HarvestKey is $harvest_key"
```

Comment:
Honestly, not really keen on printing potential private keys on the terminal.

Reply:
oh right... i think  i better remove it since i initially added it for checking purpose.